administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -172,6 +172,10 @@ COM_JOOMLAUPDATE_VIEW_UPDATE_FINALISE_HEAD_DESC="To complete the update Process
 COM_JOOMLAUPDATE_VIEW_UPDATE_PERCENT="Percent complete"
 COM_JOOMLAUPDATE_VIEW_UPLOAD_CAPTIVE_INTRO_BODY="Make sure that the update file you have uploaded comes from the official Joomla download page. Afterwards, please confirm that you want to install it by re-entering the login information for your site &quot;%s&quot; below."
 COM_JOOMLAUPDATE_VIEW_UPLOAD_CAPTIVE_INTRO_HEAD="Are you sure you want to install the file you uploaded?"
+COM_JOOMLAUPDATE_VIEW_UPLOAD_ERROR_FULLINSTALLATION="The ZIP file you uploaded looks like the Full Package of Joomla! which is only for creating new sites. You can only use the \"Upgrade Package (.zip)\" to update your site."
+COM_JOOMLAUPDATE_VIEW_UPLOAD_ERROR_NOTZIP="You need to upload the \"Upgrade Package (.zip)\" file of the Joomla version you want to upgrade to. You are trying to upload a file which is not in the ZIP format. The Joomla Update does not support the upgrade package files with .tar.gz and .tar.bz2 extensions."
```

I think it would be better to front load the error. So
```suggestion
COM_JOOMLAUPDATE_VIEW_UPLOAD_ERROR_NOTZIP="You are trying to upload a file which is not in the ZIP format. The Joomla Update does not support the upgrade package files with .tar.gz and .tar.bz2 extensions."
```
and if you really want to have that extra sentence then put it at the end
```suggestion
COM_JOOMLAUPDATE_VIEW_UPLOAD_ERROR_NOTZIP="You are trying to upload a file which is not in the ZIP format. The Joomla Update does not support the upgrade package files with .tar.gz and .tar.bz2 extensions. You need to upload the \"Upgrade Package (.zip)\" file of the Joomla version you want to upgrade to. "
```

---

I will keep the second version. I agree that front-loading is better and I am also wary that telling people what _not_ to do without telling what they _should_ be doing instead is perceived as unhelpful.